### PR TITLE
Update repo to 4.27

### PR DESCRIPTION
**Pull request by andygup**
Updated README and CHANGELOG.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 **https://js.arcgis.com/next**
 
-Welcome to the next version of the ArcGIS Maps SDK for JavaScript (4.26). This version of the SDK is currently in development and will be updated frequently until the production release in February 2023. Try out new features, perform regression testing, and get a feel for the next release - then share your feedback with the development team. Please make sure that bugs and enhancement requests are specific to https://js.arcgis.com/next or `@arcgis/core@next` and not applicable to a released version.
+Welcome to the next version of the ArcGIS Maps SDK for JavaScript (4.27). This version of the SDK is currently in development and will be updated frequently until the production release in February 2023. Try out new features, perform regression testing, and get a feel for the next release - then share your feedback with the development team. Please make sure that bugs and enhancement requests are specific to https://js.arcgis.com/next or `@arcgis/core@next` and not applicable to a released version.
```
Comment (lboyd93) [inline README.md:7]:
```suggestion
Welcome to the next version of the ArcGIS Maps SDK for JavaScript (4.27). This version of the SDK is currently in development and will be updated frequently until the production release in June 2023. Try out new features, perform regression testing, and get a feel for the next release - then share your feedback with the development team. Please make sure that bugs and enhancement requests are specific to https://js.arcgis.com/next or `@arcgis/core@next` and not applicable to a released version.
```

**Review by lboyd93 — APPROVED** [review]
Looks good!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,127 +1,32 @@
...
-| `PrintViewModel.scaleEnabled` | Use [TemplateOptions.scaleEnabled](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Print-TemplateOptions.html#scaleEnabled) or [PrintTemplate.scalePreserved](https://developers.arcgis.com/javascript/latest/api-reference/esri-rest-support-PrintTemplate.html#scalePreserved) instead | 4.22 |
-| `SearchViewModel.defaultSymbol` | Use [SearchViewModel.defaultSymbols](https://developers.arcgis.com/javascript/latest/api-reference/esri-widgets-Search-SearchViewModel.html#defaultSymbols) instead | 4.22 |
-| `StreamLayerView.effect` | Use [StreamLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-StreamLayerView.html#featureEffect) instead | 4.22 |
-| `WFSLayerView.effect` | Use [WFSLayerView.featureEffect](https://developers.arcgis.com/javascript/latest/api-reference/esri-views-layers-WFSLayerView.html#featureEffect) instead | 4.22 |
+| `esri/widgets/Slice/SlicePlane` | Use [esri/analysis/SlicePlane](https://developers.arcgis.com/javascript/latest/api-reference/esri-analysis-SlicePlane.html) instead | 4.23 |
```
Comment (annelfitz) [inline CHANGELOG.md:23]: I think this whole table can be deleted until there are breaking changes introduced.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,127 +1,32 @@
...
-- BUG-000154321: Fixed an issue where LineOfSightViewModel `observer` and `targets` returned z-values in varying elevation modes. Now all z-positions are returned as absolute values.
-- BUG-000154436: Fixed an issue within the FeatureTable widget where hiding a child column contained within a grouped column would not work correctly. This has been fixed and these contained columns can be hidden when programatically setting their visibility to `false`.
-- Esri Community - 1241411: Fixed an issue where an AbortError thrown when deleting a layer after changing its definitionExpression.
-- Fixed an issue where MediaContent charts in a popup displaying information from a relationship were only showing one related feature.
-- Fixed an issue where two `queryAttachments` requests were sent to the service when selecting a feature that had AttachmentsContent in its Popup.
-- ENH-000152699: Added support for [ImageryTileLayer](/api-reference/esri-layers-ImageryTileLayer.html) referencing a tiled image services with LODs that are not power of two.
-- ENH-000155208: Enhanced TileLayer to highlight the selected feature when displaying its Popup.
+* TBD
 
 ## Deprecations
```
Comment (annelfitz) [inline CHANGELOG.md:32]: should we add the View popup deprecation here??

**Review by annelfitz — CHANGES_REQUESTED** [review]
Just a few minor changes. Thanks!

File: CHANGELOG.md (hunk 2 above)
Comment (andygup) [reply CHANGELOG.md:23]: 👍

File: CHANGELOG.md (hunk 3 above)
Comment (andygup) [reply CHANGELOG.md:32]: Good call, adding it.

Comment (andygup) [reply CHANGELOG.md:32]: Updated, thanks.

**Pull request merged**